0-ad-history-mod-package/simulation/templates/units/han/support_trader.xml

```diff
@@ -8,6 +8,14 @@
     <Civ>han</Civ>
     <SpecificName>Shāngrén</SpecificName>
     <Icon>units/han/support_trader.png</Icon>
+    <History>
+        In the Han dynasty's early days, the Xiognu, a hostile people inhabiting the lands north of the Han empire, were repeatedly plundering the border regions. The Xiognu were nomads and rode horses, which made them extremely difficult to fight against. For that reason, the Han saw no choice but to pay tribute to the Xiognu in order to make peace. However, despite the tribute payments, the Xiognu kept on pillaging. And thanks to the Xiognu and other natural barriers, the Han were almost completely cut off from the Western part of the world.
```

Comment:
Usually in English you want to keep appositive phrases short, so the reader doesn't forget what they were reading originally.

Also, I don't think the last sentence is necciary for the paragraph.

```suggestion
        In the Han dynasty's early days, the people on its northern borders were being raided by the Xiognu. The Xiognu were nomads who lived and fought on their horses. Calvary been used as a primary force in ancient China, and so the Han were unprepared to fight back. And so, the Han saw no choice but to pay tribute to the Xiognu in order to  try to make peace. However, despite the tribute payments, the Xiognu kept on pillaging.